src/component/menubutton.jsx

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 import { connect } from "react-redux";
 
-class Button extends React.Component {
+export class Button extends React.Component {
```

Comment:
Does this need to be a class? It seems like you might get by with a stateless functional component here.